.travis.yml

```diff
@@ -4,12 +4,15 @@ python:
   - '3.5'
   - '3.4'
 sudo: required
+before_install:
+  - pip install pip==18.0
+  - pip install pipenv
 install:
   - make build
-  - pip install codecov
+  - pipenv install codecov
+  - pip install pathlib2
```

Comment:
codecov is only used by the PR process, so it's not worth adding to the pipfile right?
The pathlib2 was added because it wasn't working against python 3.4.  On reflection I should've probably added it to the pipfile instead.

Reply:
I would suggest that both should be dev dependencies. 

If codecov is being run in CI, then it should be possible to run locally too I think.

Remember that the Pipfile dependencies don't affect the actual package requirements.